tests/utils/misc/misc.go

```diff
@@ -21,8 +21,29 @@ import (
 	"time"
 )
 
+type vmstatus func(string) string
```

Comment:
1. renamed to `getVMPowerStatus`
2. that's a great idea .. we may explore more to address as part of #1301